app/models/host_aggregate.rb

```diff
@@ -25,6 +25,90 @@ class HostAggregate < ApplicationRecord
 
   virtual_total :total_vms, :vms
 
+  def self.create_aggregate_queue(userid, ext_management_system, options = {})
+    task_opts = {
+      :action => "creating Host Aggregate for user #{userid}",
+      :userid => userid
+    }
+    queue_opts = {
+      :class_name  => ext_management_system.class.class_by_ems("HostAggregate"),
+      :method_name => 'create_aggregate',
+      :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
NOTE these really don't need to be HIGH_PRIORITY but I didn't want to move and modify in the same change